<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumAnalysisDialogClass</class>
 <widget class="QDialog" name="VolumAnalysisDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_main">
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_body">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="title">
            <string>Thickness</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                  <pointsize>9</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Min Thickness</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DoubleSpinBoxControl" name="doubleSpinBox_minthick">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                  <pointsize>9</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="keyboardTracking">
                 <bool>true</bool>
                </property>
                <property name="maximum">
                 <double>5.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.030000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_minthick">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="title">
            <string>Overhang</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QCheckBox" name="checkBox_overhang">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Display Overhang</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                  <pointsize>9</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Degree</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="SpinBoxControl" name="spinBox_overhang">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                  <pointsize>9</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="maximum">
                 <number>90</number>
                </property>
                <property name="singleStep">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_overhang">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="maximum">
               <number>90</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <property name="rightMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_done">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>맑은 고딕</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Done</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBoxControl</class>
   <extends>QDoubleSpinBox</extends>
   <header>CommonWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBoxControl</class>
   <extends>QSpinBox</extends>
   <header>CommonWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doubleSpinBox_minthick</tabstop>
  <tabstop>horizontalSlider_minthick</tabstop>
  <tabstop>checkBox_overhang</tabstop>
  <tabstop>spinBox_overhang</tabstop>
  <tabstop>horizontalSlider_overhang</tabstop>
  <tabstop>pushButton_done</tabstop>
 </tabstops>
 <resources>
  <include location="rodinP.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_done</sender>
   <signal>clicked()</signal>
   <receiver>VolumAnalysisDialogClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_overhang</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_overhang</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_overhang</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_overhang</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
